recipes/libgcrypt/libgcrypt.inc

```diff
@@ -23,13 +23,6 @@ EXTRA_OECONF = "--without-pth --disable-asm ${EXTRA_OECONF_CAPABILITIES}"
 EXTRA_OECONF_CAPABILITIES = "--without-capabilities"
 EXTRA_OECONF_CAPABILITIES:USE_libgcrypt_capabilities = "--with-capabilities"
 
-# libgcrypt.pc is added locally and thus installed here
```

Comment:
I'm a bit skeptical if this can just be dropped without causing breakage, surely someone added this to be able to pick up libgcrypt settings through pkg-config. Is it installed by default now?